app/controllers/hyrax/batch_ingest/batches_controller.rb

```diff
@@ -19,6 +19,7 @@ def new
       end
 
       def create
+        @batch.uploaded_filename = params['batch']['batch_source'].original_filename
```

Comment:
Is this needed because the upload process may rename the file that gets stored in `source_location`?  Do you mind throwing in a comment somewhere that differentiates `uploaded_filename` and `source_location`?  Maybe renaming `uploaded_filename` to `batch_filename` would be clearer for non-upload use cases?

Reply:
Yes. In our case, we uploaded zip files, so those get unzipped to a tmp location which is stored in `source_location`. This value is of little use to the end user. But the original filename is.

I wasn't sure if this applied in non-upload cases.